modules/sdk-core/src/bitgo/utils/tss/baseTSSUtils.ts

```diff
@@ -0,0 +1,132 @@
+import { IRequestTracer } from 'modules/sdk-core/src/api';
+import { SerializedKeyPair } from 'openpgp';
+import { KeychainsTriplet, IBaseCoin } from '../../baseCoin';
+import { BitGoBase } from '../../bitgoBase';
+import { Keychain } from '../../keychain';
+import { getTxRequest } from '../../tss';
+import { IWallet } from '../../wallet';
+import { MpcUtils } from '../mpcUtils';
+import * as _ from 'lodash';
+import { ITssUtils, PrebuildTransactionWithIntentOptions, SignatureShareRecord, TxRequestVersion } from './baseTypes';
+
+/**
+ * BaseTssUtil class which different signature schemes have to extend
+ */
+export default class BaseTssUtils<KeyShare, TxRequest>
+  extends MpcUtils
+  // eslint-disable-next-line prettier/prettier
```

Comment:
Strange eslint error to give a new line but after giving a newline, it prompts to revert back to the old and gets the same error again, so disabled eslint check